src/components/documentation/components/card/code-snippets/card-with-image-and-text.js

```diff
@@ -0,0 +1,16 @@
+const cardWithImageAndText = `<div class="card">                
```

Comment:
try to have card size such that on laptop 4 cards are displayed in one row.  it would look better